_summaries/glorot2011deep_1.md

```diff
@@ -1,19 +1,77 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{glorot2011deep}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: ishank-arora
+score: 9
 ---
 
 TODO: Summarize the paper:
 * What is the core idea?
+
+  * First paper (I believe) to prove using rectifier function ($$max(0,  x)$$) better than $$\textit{logistic sigmoid}$$ or $$\textit{hyperbolic tangent}$$ activation functions. Approaches this from the field of computational neuroscience. 
+
+  * Proposes the use of rectifying non-linearities as alternatives to the hyperbolic tangent or sigmoid in deep artificial neural networks, in addition to using an $$L_1$$ regularizer on the activation values to promote **sparsity** and prevent potential numerical problems with unbounded activation. 
+
+  * rectifier function brings together the fields of computational neuroscience and machine learning.
+
+  * Rectifying neurons are a better model of biological neurons than hyperbolic tangenet networks.
+
+  * There are gaps between computational neuroscience models and machine learning models. Two main gaps are:
+
+    * Estimated percentage of neurons active in the brain at the same time is between 1 and 4%. Ordinary feedforward neural nets (without additional regularization such as an $$L_1$$ penalty) do not have this property.
+      * Ex: the sigmoid activation "has a steady state regime around $$\frac{1}{2}$$ , therefore, after initializing with small weights, all neurons fire at half their satura- tion regime. This is biologically implausible and hurts gradient-based optimization."
+    * Deep learning and neural networks literature (up to this point in 2011) most commonly used linear activation functions while biological neuron models use non-linear activation functions. (see figure 1)
```

Comment:
This is not true. Both use non-linear activation functions, the paper just states that they use different ones (LIF vs. sigmoid/tanh)